### ISSUE TITLE
Missing exif info for a particular image

### ISSUE THREAD

**Issue by xach** [xach/zpb-exif#2]
![dsc01950](https://cloud.githubusercontent.com/assets/16639/3866431/795ff4b6-1fd0-11e4-8c47-b11283a75f79.JPG)

From Jens Teich:

Hi Zach,

there is again a jpg where zpb-exif does not find all exif information. I attached the image.

This time I dug a bit into the problem and found "Unknown Tag" markers in the image IFD:

```
#(#<IFD-ENTRY "(Unknown Tag)" #x8825/4/1 2009D337>
  #<IFD-ENTRY "(Unknown Tag)" #x8769/4/1 2009D277>
  #<IFD-ENTRY "Make" #x10F/2/7 2009D1B7>
  #<IFD-ENTRY "Model" #x110/2/10 2009D0E7>
  #<IFD-ENTRY "DateTime" #x132/2/20 2009D013>
  #<IFD-ENTRY "Orientation" #x112/3/1 2009CF37>)
```

These are pointers to other IFDs obviously. #x8825 is GPS IFD and #x8769 is Exif IFD.

These entries ruin the bisect algorythm. I made a fix which replaces the bisect with a dumb brute force approach. This works for me for the moment.

Maybe you have a better idea?

**Comment by fourier** [comment]
Just verified, issue is fixed with pull request #7 . Could you please @xach contact the reporter so he can get an update?

**Comment by xach** [comment]
Ok, done.
